python/cudf_polars/cudf_polars/experimental/parallel.py

```diff
@@ -226,9 +226,28 @@ def task_graph(
     return graph, (get_key_name(ir), 0)
 
 
+def _register_serialize():
+    """Register Dask/cudf-polars serializers in calling process."""
+    from cudf_polars.experimental.dask_serialize import register
+
+    register()
+
+
 def evaluate_dask(ir: IR) -> DataFrame:
     """Evaluate an IR graph with Dask."""
-    from dask import get
+    from dask import get as _get
+    from distributed import get_client
+
+    _register_serialize()
```

Comment:
Why redirect to `_register_serialize` instead of importing/calling `register` directly?

Reply:
No particular reason, we could just move the import to the top and call `register()` here as well as in `client.run`/`client.run_on_scheduler` down below.